### DISCUSSION TITLE
EXPLORER PATCHER  - VALINET

### DISCUSSION THREAD

**Discussion by madnwise** [community/community#38347] [Projects and Issues]
I am Using the latest EXPLORER PATHCER - VALINE
After the recent updating of window 11, the start button had freeze. I had to uninstall the EP and then the start button  started working. I think there is some issue and needs to be sorted out.  

Hoping for immediate rectfication

MNW

**Comment by ossanna16** [comment]
Hi @madnwise,
Thanks for being a part of the GitHub Community! We're glad you're here.

I wanted to jump in and mention that if you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Explorer Patcher. It's definitely possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
